lib/maintenance_smelt.pm

```diff
@@ -14,12 +14,14 @@ use Mojo::UserAgent;
 use base "Exporter";
 use Exporter;
 
+my $smelt_url = get_var("SMELT_URL", "https://smelt.suse.de");
+
```

Comment:
Please add this to variables.md

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20520/files#diff-eb176a275e164bf44779feb951ab9bbcbb60959e2b5227a4e23101962d69299eR295